generators/client/templates/vue/src/main/webapp/app/admin/user-management/user-management-view.vue.ejs

```diff
@@ -53,10 +53,11 @@
             </ul>
           </dd>
         </dl>
-        <router-link tag="button" type="submit"
-                :to="{name: '<%= jhiPrefixCapitalized %>User'}"
-                class="btn btn-info">
+        <router-link custom v-slot="{ navigate }"
+                :to="{name: '<%= jhiPrefixCapitalized %>User'}">
+          <button @click="navigate" type="submit" class="btn btn-info">
```

Comment:
Should it be really a `submit` button and not just a `button` (was already before the case)?

Reply:
Yes, it was already there, but seems it is not required to be a submit, as its not even a form out there and its just a back functionality. Removed submit in latest commit.